types/documentdb/index.d.ts

```diff
@@ -126,12 +129,8 @@ interface AbstractMeta extends UniqueId {
 
 /** Represents a custom document for storing in DocumentDB  */
 interface NewDocument<TContent> extends UniqueId {
```

Comment:
`doc` was commented out, so was this ever a generic type? I'm inclined to remove `doc` and the `TContent` generic, since a DDB document may define any number of custom top-level properties. It might make more sense to apply the `TContent` generic to methods that accept/return `Document` types.